### PR TITLE
Fix trainer arg for latest Nemo compatibility

### PR DESCRIPTION
**Pull request by borisfom**

### CODE REVIEW

File: nemo2riva/convert.py [hunk 1]
```diff
@@ -35,15 +36,21 @@ def Nemo2Riva(args):
     logging.info("Restoring NeMo model from '{}'".format(nemo_in))
     # Create a PL trainer object which is required for restoring Megatron models
     cfg_trainer = TrainerConfig(
-        accelerator="auto",
+        accelerator='gpu',
+        strategy="ddp",
```
Comment (blisc) [inline nemo2riva/convert.py:40]: Do we need to set this? Does it crash if set to auto?

Comment (svenchilton) [reply nemo2riva/convert.py:40]: @blisc: @borisfom implemented this change to fix an error which I kept encountering when running `nemo2riva`, namely 
```
ValueError: You selected an invalid accelerator name: `accelerator=TrainerConfig(logger=False, callbacks=None, default_root_dir=None, gradient_clip_val=0, num_nodes=1, enable_progress_bar=True, overfit_batches=0.0, check_val_every_n_epoch=1, fast_dev_run=False, accumulate_grad_batches=1, max_epochs=1000, min_epochs=1, max_steps=-1, min_steps=None, limit_train_batches=1.0, limit_val_batches=1.0, limit_test_batches=1.0, val_check_interval=1.0, log_every_n_steps=50, accelerator='auto', sync_batchnorm=False, precision=32, num_sanity_val_steps=2, profiler=None, benchmark=False, deterministic=False, use_distributed_sampler=True, detect_anomaly=False, plugins=None, limit_predict_batches=1.0, gradient_clip_algorithm='norm', max_time=None, reload_dataloaders_every_n_epochs=0, devices=None, strategy=None, enable_checkpointing=False, enable_model_summary=True, inference_mode=True, barebones=False)`. Available names are: auto, cpu, cuda, hpu, ipu, mps, tpu
```

Comment (borisfom) [reply nemo2riva/convert.py]: @blisc : auto works as well. Reverted that part.

Comment (svenchilton) [reply nemo2riva/convert.py]: @blisc, @borisfom, oh, I guess the change in the parameters of TrainerConfig wasn't necessary. Thanks for fixing my issue in any event, Boris.

**Review by blisc — APPROVED** [review]

**Pull request merged**